MobileBuy/sample/src/main/java/com/shopify/sample/activity/CheckoutActivity.java

```diff
@@ -53,7 +54,15 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setTitle(R.string.checkout);
-        setContentView(R.layout.checkout_activity);
+
+        OptimizelyClient optimizelyClient = getSampleApplication().getOptimizelyManager().getOptimizely();
+        Variation product_sorting_variation = optimizelyClient.getVariation("checkout_activity_layout",
```

Comment:
But: I'd make the experiment key a little more realistic like checkout_cta since you are experimenting on the checkout call to action